tests/test_declarations/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -93,22 +93,26 @@ it(\`handles
   console.log("hello!");
 });
 
-test("does something really long and complicated so I have to write a very long name for the test", done => {
+test("does something really long and complicated so I have to write a very long name for the test", (
+  done
```

Comment:
It is testing that the string literal doesn't break to the next line. There are special cases for `test`/`it`, etc.